src/site/layouts/media_list_images.drupal.liquid

```diff
@@ -4,109 +4,92 @@
-                <li className="vads-u-display--flex vads-u-margin-bottom--2">
+                    <article>
+                        <!-- Title -->
+                        <h1>{{ title }}</h1>
 
-                  {% assign alt = mediaImage.entity.image.alt %}
-                  {% if alt == mediaImage.entity.entityLabel %}
-                    {% assign alt = '' %}
-                    {{ testing }}
-                  {% endif %}
+                        <!-- Intro -->
+                        <div class="va-introtext">{{ fieldIntroTextLimitedHtml.processed }}</div>
 
-                  <div class="vads-l-grid-container vads-u-padding-x--0 vads-u-margin-y--2">
-                    <div class="vads-l-row">
-                      <div class="vads-l-col--2">
-                        <img class="vads-u-border--1px vads-u-border-color--gray-light" alt="{{ alt }}"
-                          height="{{ mediaImage.entity.image.height }}" width="{{ mediaImage.entity.image.width }}"
-                          src="{{ mediaImage.entity.image.url }}" />
-                      </div>
-                      <div class="vads-l-col--10 vads-u-padding-left--2">
-                        <h2 class="vads-u-font-size--h3">{{ mediaImage.entity.entityLabel }}</h2>
-                        {% if mediaImage.entity.fieldDescription %}
-                          <p>{{ mediaImage.entity.fieldDescription }}</p>
+                        <!-- Alert -->
+                        {% if fieldAlertSingle %}
+                            {% include "src/site/paragraphs/alert_single.drupal.liquid" with entity = fieldAlertSingle.entity %}
                         {% endif %}
-                      </div>
-                    </div>
-                  </div>
-                </li>
-              {% endfor %}
-            </ul>
 
+                        <!-- Buttons -->
+                        <div class="vads-u-margin-y--3">
+                            {% for fieldButton in fieldButtons %}
+                                {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
+                            {% endfor %}
+                        </div>
+                        <!-- Section Header -->
+                        {% if fieldMediaListImages.entity.fieldSectionHeader %}
+                            {% assign image_title_tag = "h3" %}
+                            {% assign image_title_classes = "vads-u-margin-y--0 medium-screen:vads-u-margin-y--1p5" %}
+                            <h2 class="vads-u-margin-bottom--4">{{fieldMediaListImages.entity.fieldSectionHeader }}</h2>
+                        {% else %}
+                            {% assign image_title_tag = "h2" %}
+                            {% assign image_title_classes = "vads-u-font-size--h3" %}
+                        {% endif %}
+                        <!-- Images List -->
+  
+                        <dl class="usa-unstyled-list">
```

Comment:
I think the images list comment should be moved down one line right?

Reply:
yuppp